cmd/prowjob/prowjob.go

```diff
@@ -26,4 +26,5 @@ var ProwjobCmd = &cobra.Command{
 func init() {
 	ProwjobCmd.AddCommand(periodicSlackReportCmd)
 	ProwjobCmd.AddCommand(createReportCmd)
+	ProwjobCmd.AddCommand(sendSlackMessageCmd)
```

Comment:
I'm thinking if we could move this command to the root command level, i.e. to `cmd/sendSlackMessage.go`, since this is just a general command and it's not tied to the prowjob. wdyt?

then it could be run as `./qe-tools send-slack-message [args]`